docs/install/upgrade.md

```diff
@@ -43,4 +43,8 @@ So, copying `notebook` and `conf` directory should be enough.
 
    ```
    bin/zeppelin-daemon.sh start
-   ```
\ No newline at end of file
+   ```
+
+# Breaking Changes
+## Upgrading from 0.6.x to 0.7.0
+- Mapping from `%jdbc(prefix)` to `%prefix` is no longer available. You can set multiple interpreters with same interpreter group.
```

Comment:
nit: How about "Instead, you can use `%[interpreter alias]` with multiple interpreter setttings on GUI", instead of "You can set multiple interpreters with same interpreter group"?